crates/common_enums/Cargo.toml

```diff
@@ -13,3 +13,4 @@ diesel = { version = "2.0.3", features = ["postgres"] }
 
 # First party crates
 router_derive = { version = "0.1.0", path = "../router_derive" }
+serde_json = "1.0.95"
```

Comment:
You should be able to move `serde_json` under the `[dev-dependencies]` table, since it's only used in tests as of now.

> Any specific reason these tests are failing?
> My changes seem unrelated to them (except formatting)

It looks like you ran a `cargo update` (or something else triggered it), which has bumped multiple dependencies and tests are breaking. You could try reverting the `Cargo.lock` to an earlier commit, move `serde_json` to dev-deps and run `cargo check`. That should update the `Cargo.lock` to a stable state.